src/server/heart-beat-db.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-const config = require('../../express.config.json');
+const config = require('../../.config.json');
 const MongoClient = require('mongodb').MongoClient;
 const url = `${config.db.url}:${config.db.port}/${config.db.databaseName}`;
```

Comment:
put every part into a variable like:

```js
const address = config.db.url;
const port = config.db.port
```

maybe create a function that can do eg `createDatabaseUrl` 😉